components/README.md

```diff
@@ -0,0 +1,19 @@
+Components
+==========
+
+This directory includes a library of generic React components to be used for creating common UI elements shared between screens and features of the WordPress dashboard.
```

Comment:
I love this. Shared UI components inside WordPress. This makes me think we should showcase those.